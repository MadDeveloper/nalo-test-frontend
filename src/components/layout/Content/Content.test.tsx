import renderer from "react-test-renderer"
import { Content } from "."

describe("Components > Content", () => {
  it("should match snapshot", () => {
    const tree = renderer
      .create(
        <div>
          <Content />
          <Content small />
          <Content fullwidth />
          <Content noHorizontalSpacingOnMobile />
          <Content noHorizontalSpacingOnTablet />
          <Content noVerticalSpacing />
        </div>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
