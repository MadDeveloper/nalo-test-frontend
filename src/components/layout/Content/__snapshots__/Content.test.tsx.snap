// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Content should match snapshot 1`] = `
<div>
  <div
    className="root"
  />
  <div
    className="root small"
  />
  <div
    className="root fullwidth"
  />
  <div
    className="root noHorizontalSpacingOnMobile"
  />
  <div
    className="root noHorizontalSpacingOnTablet"
  />
  <div
    className="root noVerticalSpacing"
  />
</div>
`;
